doc/nrf/app_dev/device_guides/nrf54l/cryptography.rst

```diff
@@ -0,0 +1,353 @@
+
+The application can use the KMU slots to store key data for their own purposes.
+Some KMU slots are reserved for current and future |NCS| use cases.
+The following table gives an overview of the KMU slots and their usage:
+
+.. list-table:: List of reserved KMU slots
+   :widths: auto
+   :header-rows: 1
+
+   * - | Reserved KMU slots
+       | (range inclusive)
+     - | NCS usage
+       |
+     - | Description
+       |
+   * - 180-182
+     - Reserved
+     - --
+   * - 183-185
+     - IKG seed
+     - 384-bit random seed to generate keys using the CRACEN IKG.
+   * - 186-225
+     - Reserved
+     - --
+   * - 226-227
+     - UROT_PUBKEY_0
+     - | Revokable firmware image key for upgradable bootloader, generation 0.
+       | ED25519 public key.
+   * - 228-229
+     - UROT_PUBKEY_1
+     - | Revokable firmware image key for upgradable bootloader, generation 1.
+       | ED25519 public key.
+   * - 230-231
+     - UROT_PUBKEY_2
+     - | Revokable firmware image key for upgradable bootloader, generation 2.
+       | ED25519 public key.
+   * - 232-241
+     - Reserved
+     - --
+   * - 242-243
+     - BL_PUBKEY_0
+     - | Revokable firmware image key for immutable bootloader, generation 0.
+       | ED25519 public key.
+   * - 244-245
+     - BL_PUBKEY_1
+     - | Revokable firmware image key for immutable bootloader, generation 1.
+       | ED25519 public key.
+   * - 246-247
+     - BL_PUBKEY_2
+     - | Revokable firmware image key for immutable bootloader, generation 2.
+       | ED25519 public key.
+   * - 248-255
+     - Reserved
+     - --
+
+CRACEN Isolated Key Generator
+=============================
+
+CRACEN's Isolated Key Generator (IKG) can derive three keys from a 384-bit seed value.
+
+IKG-generated keys are also called special hardware keys.
+These keys are typically not meant for application use.
+They are not accessible by any CPU, but they can be used for cryptographic operations by CRACEN, provided directly to the cryptographic engine as a hardware signal.
+IKG keys are not retained, and have to be regenerated for every CRACEN power cycle.
+
+The 384-bit seed value is generated automatically during the first boot of the device using the CRACEN Random Number Generator (RNG), and then stored in the KMU.
+Before the keys can be generated, the seed is pushed by the KMU to the SEED register, and then validated.
+Generating keys without the valid seed will fail.
+
+IKG keys are also accessed using the standard PSA Crypto APIs, and are referenced by special built-in key IDs.
+
++-----------------+-------------------------------------+---------------------------------------------------+
+| Key type        | Key ID                              | Description                                       |
++=================+=====================================+===================================================+
+| ECC secp256r1   | ``CRACEN_BUILTIN_IDENTITY_KEY_ID``  | Used for signing/verification.                    |
```

Comment:
This is changing due to PR: https://github.com/nrfconnect/sdk-nrf/pull/17844

The new proposed name is `CRACEN_BUILTIN_XXXX_ID`